src/platform/storage/zephyr/nvs/storage.c

```diff
@@ -163,9 +197,149 @@ mender_storage_init(void) {
                      mender_storage_nvs_handle.sector_count,
                      (mender_storage_nvs_handle.sector_count - 1) * mender_storage_nvs_handle.sector_size);
 
+#ifdef CONFIG_MENDER_DEPLOYMENT_LOGS
+    /* Initialize the Flash Circular Buffer (FCB) for deployment logs. */
+
+    /* just some fixed magic value the FCB implementation uses to mark
+       in-use/erased/... sectors */
+    depl_logs_buffer.f_magic       = 0x2AABEE35;
```

Comment:
The magic seems to be completely user defined. Consider using a Kconfig setting for it

Reply:
Do you think people would like to set it themselves? Why?

Reply:
I don't have a reason. Is just that it looks very esoteric to have here a hard-coded value. What if we have it at the beginning of the file with a define at least?